display.h

```diff
@@ -80,3 +78,43 @@ void showGameInstructions() {
     delay(60);
     printw("\n\n\n\n\n\t///////////////////////////////////////////////////////////////////////////////////////////");
 }
+
+void showGameDifficultyOptions0() {
```

Comment:
This name isn`t very explained, you can change for : showGameDifficultyOptionsEasy

Reply:
Well catch, thanks!

Reply:
Done!